.github/workflows/windows.yml

```diff
@@ -46,8 +46,9 @@ jobs:
           
     - name: Build with cmake
       run: |
-        cmake -B ${{github.workspace}}/build  -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
-        cmake --build ${{github.workspace}}/build -j`nproc` --config ${{env.BUILD_TYPE}}
+        ls -la
+        cmake.exe -B D:\a\qucsator_rf\qucsator_rf\build  -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
```

Comment:
Cmake from MSYS2 requires explicit generator specification. Unspecified generator may not work.